cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -96,11 +97,11 @@
     public static final String ENV_VAR_STRIMZI_JAVA_SYSTEM_PROPERTIES = "STRIMZI_JAVA_SYSTEM_PROPERTIES";
     public static final String ENV_VAR_STRIMZI_GC_LOG_ENABLED = "STRIMZI_GC_LOG_ENABLED";
 
-    public static final String ANNO_STRIMZI_IO_DELETE_CLAIM = Annotations.STRIMZI_DOMAIN + "/delete-claim";
+    public static final String ANNO_STRIMZI_IO_DELETE_CLAIM = Annotations.STRIMZI_DOMAIN + "delete-claim";
     /** Annotation on PVCs storing the original configuration (so we can revert changes). */
-    public static final String ANNO_STRIMZI_IO_STORAGE = Annotations.STRIMZI_DOMAIN + "/storage";
+    public static final String ANNO_STRIMZI_IO_STORAGE = Annotations.STRIMZI_DOMAIN + "storage";
     @Deprecated
-    public static final String ANNO_CO_STRIMZI_IO_DELETE_CLAIM = "cluster.operator.strimzi.io/delete-claim";
+    public static final String ANNO_CO_STRIMZI_IO_DELETE_CLAIM = ClusterOperator.STRIMZI_CLUSTER_OPERATOR_DOMAIN + "/delete-claim";
```

Comment:
I do not see any changes in `ClusterOperator`. Which I guess means that the `strimzi.io` is still hardcoded there as:

```
public static final String STRIMZI_CLUSTER_OPERATOR_DOMAIN = "cluster.operator.strimzi.io";
```

which should be probably changed? Or did I missed this change somewhere in this PR?

Reply:
It is still hardcoded there, the string itself is used so infrequently and its relationship to the other strimzi labels was slightly unclear to me.
For instance I didn't know if this string was truly dependent on the STRIMZI_DOMAIN, e.g.
```
public static final String STRIMZI_CLUSTER_OPERATOR_DOMAIN = "cluster.operator." + Labels.STRIMZI_DOMAIN;
```

if you are happy to confirm the relationship is as above, then I am happy to modify it. :)

Reply:
Just to add it is only in used in one place
```
    @Deprecated
    public static final String ANNO_CO_STRIMZI_IO_DELETE_CLAIM = "cluster.operator.strimzi.io/delete-claim";
```
I also noted this string is deprecated, so though we could change it, I would argue that since it is probably going to be removed, defining its relationship to the rest of the Strimzi Labels might not be needed. Do you have any thoughts on this?